### DISCUSSION TITLE
project sdk ="Microsoft.Net.Sdk.Razor" build failed

### DISCUSSION THREAD

**Discussion by ChisterWu** [AvaloniaUI/Avalonia#11703] [General]
When  I  change Project Sdk="Microsoft.NET.Sdk.Razor" on android project, It build failed. How can i do for it?
This is my BlazorWebView Sample for avaloniaUi.
https://github.com/MicroSugarDeveloperOrg/Avalonia.WebView.Sample

**Comment by maxkatz6** [comment]
Why do you mix Microsoft.NET.Sdk.Razor with normal SDK? AFAIK Microsoft.NET.Sdk.Razor is for asp.net web application.

**Reply by ChisterWu** [reply]
I have developed a complete webview for avalonia, and I want add support for blazor，it need change the sdk = "Microsoft.NET.Sdk.Razor",  it works ok in ios desktop ,  but andioid platform can't compile
